### PR TITLE
feat: add service type for karpor server

### PR DESCRIPTION
**Pull request by gshilei**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):
- feat: https://github.com/KusionStack/karpor/issues/548

#### 2. What is the scope of this PR (e.g. component or file name):
- charts/karpor/

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):
- feat: add service type for karpor server

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- N

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by gshilei** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ruquanzhao** [comment]
cc @elliotxx

File: charts/karpor/README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ The Karpor Server Component is main backend server. It itself is an `apiserver`,
 | server.port | int | `7443` | Port for karpor server. |
 | server.replicas | int | `1` | The number of karpor server pods to run. |
 | server.resources | object | `{"limits":{"cpu":"500m","ephemeral-storage":"10Gi","memory":"1Gi"},"requests":{"cpu":"250m","ephemeral-storage":"2Gi","memory":"256Mi"}}` | Resource limits and requests for the karpor server pods. |
+| server.serviceType | string | "ClusterIP"                                                                                                                                | Service type for the karpor server. The available type values list as ["ClusterIP"、"NodePort"、"LoadBalancer"]. |
```
Comment (ruquanzhao) [inline charts/karpor/README.md:79]: nit: first  "ClusterIP" ==>  &#96;"ClusterIP" &#96;

**Comment by elliotxx** [comment]
@gshilei The Chart version in `charts/karpor/README.md` `0.6.11` also needs to be updated to `0.6.12`

**Comment by gshilei** [comment]
> @gshilei The Chart version in `charts/karpor/README.md` `0.6.11` also needs to be updated to `0.6.12`

Fix

**Comment by ruquanzhao** [comment]
@gshilei 
image
please take a look

**Comment by gshilei** [comment]
> @gshilei image please take a look

ok

**Review by ruquanzhao — APPROVED** [review]

**Comment by ruquanzhao** [comment]
Thanks ! @gshilei  🚀🚀🚀

**Pull request merged**